.github/workflows/stale.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    - cron: '00 19 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          stale-issue-message: |
+            This issue is marked as stale because author feedback has been requested for 30 days with no response. Please respond within 14 days or this issue will be closed due to inactivity.
+          close-issue-message: |
+            This issue was closed due to inactivity. If you can still reproduce this bug, please comment with the requested information, detailed steps to reproduce the problem, or any other notes that might help in the investigation.
+          start-date: '2021-01-06T00:00:00Z'
+          days-before-stale: 30
+          days-before-close: 14
```

Comment:
Is this 14 days between "stale" and "closed"? So 44 days total?

Reply:
Yeah, wasn't thinking about a nice number. Rather, "one month until marked as stale", and "two weeks from then to close"